### PR TITLE
feat(aws-eks-iam-role): add permissions_boundary to eks-iam-role

### PR DESCRIPTION
**Pull request by topikachu**
## what
* add permissions_boundary to aws_iam_role

## why
* Our org requires all IAM role has permissions_boundary

## references
* https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role#permissions_boundary

### CODE REVIEW

**Comment by topikachu** [comment]
@dylanbannon Would you please take a look at this change?

File: variables.tf [hunk 1]
```diff
@@ -40,3 +40,9 @@ variable "eks_cluster_oidc_issuer_url" {
     error_message = "The eks_cluster_oidc_issuer_url value must have a value."
   }
 }
+
+variable "permissions_boundary" {
+  type        = string
+  description = "If provided, IAM role will be created with this permissions boundary attached"
```
Comment (nitrocode) [inline variables.tf]:
```suggestion
  description = "ARN of the policy that is used to set the permissions boundary for the role."
```

**Review by nitrocode — APPROVED** [review]

**Comment by topikachu** [comment]
@nitrocode  Thanks for your approval.  
Can somebody merge this change, because I have another one of https://github.com/cloudposse/terraform-aws-helm-release depends on this.

**Review by Gowiem — APPROVED** [review]

**Comment by Gowiem** [comment]
@topikachu Thanks for the contribution! Released as https://github.com/cloudposse/terraform-aws-eks-iam-role/releases/tag/1.1.0

**Pull request merged**